cloud_governance/policy/policy_operations/aws/zombie_non_cluster/zombie_non_cluster_polices.py

```diff
@@ -33,7 +33,7 @@ def run(self):
                             for policy_dict in policy_result:
                                 policy_dict['region_name'] = self._region
                                 policy_dict['account'] = self._account
-                                self._es_operations.upload_to_elasticsearch(data=policy_dict, index=self._es_index)
+                                self._es_operations.upload_to_elasticsearch(data=policy_dict.copy(), index=self._es_index)
```

Comment:
What the purpose of copy ?

Reply:
I do not want to pass ref value, I need to send the values directly.

Reply:
yes

Reply:
Are u sure its documentation issue ?

Reply:
Added bug label too

Reply:
OK